axelrod/tests/integration/test_tournament.py

```diff
@@ -45,7 +46,7 @@ def setUpClass(cls):
     def test_big_tournaments(self, tournament):
         """A test to check that tournament runs with a sample of non-cheating
         strategies."""
-        filename = "test_outputs/test_tournament.csv"
+        filename = axl_filename("test_outputs/test_tournament.csv")
```

Comment:
Also with pathlib, here you could do something like `filename = axl_filename() / "test_outputs" / "test_tournament.csv`, assuming that axl_filename with no arguments just returns the right directory.

Reply:
I like this idea to get it platform independent.  But actually the cast to string needs to happen before applying the / operator.  So this format wouldn't work.

What I did instead was take multiple arguments so that you could do:
axl_filename("test_outputs", "test_tournament.csv")
Then the function would apply the / operator inside.